homeassistant/components/freebox/router.py

```diff
@@ -159,14 +160,21 @@ async def _update_disks_sensors(self) -> None:
 
     async def _update_raids_sensors(self) -> None:
         """Update Freebox raids."""
-        # None at first request
+        if not self.supports_raid:
+            return
+
         try:
             fbx_raids: list[dict[str, Any]] = await self._api.storage.get_raids() or []
         except HttpRequestError:
-            _LOGGER.warning("Unable to enumerate raid disks")
-        else:
-            for fbx_raid in fbx_raids:
-                self.raids[fbx_raid["id"]] = fbx_raid
+            self.supports_raid = False
+            _LOGGER.warning(
+                "Router %s API does not seem to support raid, will not enumerate further",
```

Comment:
I'd keep it short and not say that we're unsure about what the case is and not talk about enumeration. We only know what the api tells us. There's no need to go further into that. The user doesn't know what enumeration means in this context. 
```suggestion
                "Router %s API does not support raid",
```

Also, I'd consider lowering the log message to info, if there's nothing wrong or something the user can do about this.